### PR TITLE
Rd 837 tokens in their own fields

### PR DESCRIPTION
**Pull request by ZephSibley**
The prefix "actual" is an unfortunate necessity in order to avoid naming clashes with the core package, which is a dependency that we can't change here.

### CODE REVIEW

File: social_django/storage.py [hunk 1]
```diff
@@ -145,6 +145,51 @@ def create_social_auth(cls, user, uid, provider):
...
+            self.actual_access_token = extra_data.pop('access_token', None)
+            self.actual_refresh_token = extra_data.pop('refresh_token', None)
+            if self.set_extra_data(extra_data):
+                self.save()
+
+    def set_extra_data(self, extra_data=None):
+        """
+        Making sure we never store the tokens in extra data
+        """
+        access_token = extra_data.pop('access_token', None)
```
Comment (BenRogersNewsome) [inline social_django/storage.py]: What happens if extra_data is None here?

**Review by BenRogersNewsome — APPROVED** [review]

File: social_django/storage.py (hunk 1 above)
Comment (ZephSibley) [reply social_django/storage.py]: We just set it to None!

Comment (BenRogersNewsome) [reply social_django/storage.py]: Will None.pop(..)  return None or create an error though?

Comment (ZephSibley) [reply social_django/storage.py]: Oh wait I misread the question, good point

**Pull request merged**